start.sh

```diff
@@ -21,4 +17,4 @@ if [ "$(ls -A /opt/snap/tasks/)" ]; then
 fi
 
 # run SNAPTELD
-exec /opt/snap/sbin/snapteld -t ${SNAP_TRUST_LEVEL} -l ${SNAP_LOG_LEVEL} -o ''
```

Comment:
snapteld will read SNAP_TRUST_LEVEL, SNAP_LOG_LEVEL and SNAP_LOG_PATH (-o) from environment variables, we dont need to pass them as cmdline args.